### PR TITLE
betty: update page

### PR DESCRIPTION
**Pull request by kbdharun**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

From #9578

### CODE REVIEW

**Review by Magrid0 — COMMENTED** [review]
is the `betty go crazy` really necessary? I don't know this command, just asking.

**Comment by kbdharun** [comment]
> is the `betty go crazy` really necessary? I don't know this command, just asking.

I have no idea, I too don't use this command, for the changes I referred to documentation + tweaked it for our syntax.

**Comment by Magrid0** [comment]
I'll do some research

**Comment by SethFalco** [comment]
I see nothing wrong with including the command. Once we reach the 8 examples we can discuss what should or shouldn't be included, and perhaps `betty go crazy` can be one of the first to go.

Until then, seems like a fun command and description. :+1:

**Review by Magrid0 — CHANGES_REQUESTED** [review]
~~I think we should remove the `betty go crazy` part, it doesn't seem very useful, seems more like that `sl` command that show a train, it's just for fun.~~

**Comment by SethFalco** [comment]
Commands here don't have to be useful, they have to show what the tool is capable of.
Not to mention, it's not our job to determine what is useful or not. Everything has a use-case to someone.

**Review by Magrid0 — APPROVED** [review]

File: pages/common/betty.md [hunk 1]
```diff
@@ -34,3 +30,7 @@
 - Drive Betty to madness:
```
Comment (mfrw) [inline pages/common/betty.md:30]: Well why not :)
🥇

**Review by mfrw — APPROVED** [review]

**Comment by mfrw** [comment]
> is the `betty go crazy` really necessary? I don't know this command, just asking.

IMHO, this adds a little humor to the pages and I feel its okay :)
Afterall I always enjoy some giggle while on the terminal.

**Pull request merged**